src/components/Section/History/Residence/ResidenceItem.jsx

```diff
@@ -332,8 +332,7 @@ export default class ResidenceItem extends ValidationElement {
         </Field>
         <Show
           when={
-            (this.props.Role || {}).value &&
-            !['Own', 'Rent', 'Military'].includes((this.props.Role || {}).value)
+            this.props.Role && this.props.Role.value === 'Other'
```

Comment:
So `this.props.Role` doesn't always have a property named `value`, but with this change that undefined result is compared to `Other`. If you put a breakpoint at this line and open up this page, you can see that happening when you create a new residence (and Role.value is not set), or you have a fresh application form.

Chrome doesn't throw an error when comparing against `undefined`, but it doesn't seem like we'd want to do comparisions like that. That's why you see this pattern all over the place, checking to see if a property is set before using it.

Reply:
What's wrong with comparing `undefined` to the string "Other"?

The conditional will be falsy and the condition is not satisfied so the explanation field will not display. I'm not aware of any risk or anti pattern surrounding this.

Reply:
@el-mapache and I talked about this pattern the other day, and would agree that the `|| {}` seems unnecessary, however would like to get @el-mapache's opinion again just in case this is catching something that I'm unaware of.

Reply:
While I can see why they used this pattern in other sections of the code, in this case it's unnecessary to repeat the `|| {}` guard multiple times, and @christopherhuii 's check is sufficient!

Reply:
One thing I would like to see is less reliance overall on hard-coded strings, but that should be examined in a separate issue